src/cogs/pin.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from discord import app_commands
+from discord.ext.commands import Cog  # pyright: ignore[reportMissingTypeStubs]
+
+if TYPE_CHECKING:
+    from discord import Interaction, Message
+
+    from bot import FISABot
+
+
+class Pin(Cog):
+    @app_commands.context_menu(name="Épingler/Désépingler")
+    async def pin(interaction: Interaction, message: Message):
+        if message.pinned():
+            await message.unpin()
+            await interaction.response.send_message("Message désépinglé.",ephemeral=True)
+        else:
+            await message.pin()
+        await interaction.response.defer()
```

Comment:
```suggestion
        	await interaction.response.defer()
```
Don't defer if the message was unpinned